### PR TITLE
Updated images

### PR DESCRIPTION
**Pull request by LEQUI000**
Added images and graphics, updated file structure for organization

### CODE REVIEW

File: _posts/dart-cuts.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Background on DART cuts"
 excerpt: "At the time of writing this, 5 cities have voted for and 2 cities have voted against a reduction in DART's funding by 25%. Here's how we got here and what's coming next."
-coverImage: "/assets/blog/dart-cuts/cover.jpg"
+coverImage: "/assets/blog/dart-cuts/dartCuts.png"
```
Comment (CurtisGreen) [inline _posts/dart-cuts.md]: everything is dash-case so I think this and the other file names should also file that

Comment (LEQUI000) [reply _posts/dart-cuts.md]: Sorry I grabbed the direct file from Connor and didn't rename it

Comment (CurtisGreen) [reply _posts/dart-cuts.md]: Still need to rename this to `dart-cuts.png`, same with some of the other images

Comment (LEQUI000) [reply _posts/dart-cuts.md]: Files were renamed in the latest commit

File: public/assets/blog/dart-cuts/cover.png
Comment (CurtisGreen) [inline public/assets/blog/dart-cuts/cover.png:1]: You have `dart-cuts.png` and `cover.png`

File: public/assets/blog/dart-cuts/sw-med-dist-parkland-0758.jpg
Comment (CurtisGreen) [inline public/assets/blog/dart-cuts/sw-med-dist-parkland-0758.jpg:1]: This is a duplicate of the myths-facts cover image, should be deleted

Comment (LEQUI000) [reply public/assets/blog/dart-cuts/sw-med-dist-parkland-0758.jpg:1]: file was deleted

**Review by CurtisGreen — APPROVED** [review]

**Pull request merged**